src/parser/tptp/tptp_antlr.h

```diff
@@ -0,0 +1,82 @@
+/******************************************************************************
+ * Top contributors (to current version):
+ *   Francois Bobot, Andrew Reynolds, Mathias Preiner
+ *
+ * This file is part of the cvc5 project.
+ *
+ * Copyright (c) 2009-2022 by the authors listed in the file AUTHORS
+ * in the top-level source directory and their institutional affiliations.
+ * All rights reserved.  See the file COPYING in the top-level source
+ * directory for licensing information.
+ * ****************************************************************************
+ *
+ * Definition of TPTP parser.
+ */
+
+#include "cvc5parser_private.h"
+#include "parser/antlr_input.h"  // Needs to go first.
```

Comment:
Not sure I understand the comment, why does it need to go first?

Reply:
This was copied :)  I don't understand it either, I guess it can be removed.